### PR TITLE
Midterm

### PR DESCRIPTION
**Pull request by Mr-Edwin**

### CODE REVIEW

**Comment by halfaro1** [comment]
@Mr-Edwin the automated tests didn't run on your code - it occurs to me that maybe it's because of the way you're submitting the pull request? are you doing it through the web or the desktop client? In any case, closing and reopening your pull request allowed me to run the automated tests. It looks like your program failed some automated tests, I think it's because of the rounding on Q1? Try fixing that.

**Comment by Mr-Edwin** [comment]
Sorry professor but I went on a week long trip to South Carolina for a robotics competition and didn't see this until today. I took the round() part out of my question 1 and now Im getting a Validation Error when I try to do a pull request.

File: q1.py [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
+
+    if .021 * balance > 10:
+        minimumPay = .021 * balance
+    else:
+        minimumPay = 10
+
+    if minimumPay > balance:
+        minimumPay = balance;
+
+    return minimumPay,1
```
Comment (halfaro1) [inline q1.py]: @Mr-Edwin what's the deal with the comma here on line 20?

Comment (Mr-Edwin) [inline q1.py]: picture

Comment (Mr-Edwin) [inline q1.py]: picture2

**Pull request closed without merging**